stacker/actions/build.py

```diff
@@ -0,0 +1,201 @@
+import logging
+
+from . import base
+from .. import exceptions, util
+from ..plan import Plan
+
+logger = logging.getLogger(__name__)
+
+
+class Action(base.BaseAction):
```

Comment:
I prefer to have the filename set the context for the class it contains.

this isn't a very strong preference so I can rename if you want.